home/management/commands/create_default_site.py

```diff
@@ -1,16 +1,33 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
-from django.contrib.sites.models import Site
+from django.contrib.sites.models import Site as DjangoSite
 from django.core.management.base import BaseCommand
+from wagtail.core.models import Site as WagtailSite
+
+from home.models import HomePage
 
 User = get_user_model()
 
 
 class Command(BaseCommand):
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--domain',
+            help='The domain on which the website is deployed'
+        )
+
     def handle(self, *args, **options):
-        _, created = Site.objects.update_or_create(pk=settings.SITE_ID, defaults={
-            'domain': 'localhost:8000',
+        domain = options.get('domain') or 'localhost:8000'
+
+        __, created = DjangoSite.objects.update_or_create(pk=settings.SITE_ID, defaults={
+            'domain': domain,
             'name': 'default'
         })
-        self.stdout.write(self.style.SUCCESS(f'Site object {"created" if created else "updated"}'))
+        self.stdout.write(self.style.SUCCESS(f'Django Site object {"created" if created else "updated"}'))
+        __, created = WagtailSite.objects.update_or_create(is_default_site=True, defaults={
+            'hostname': domain,
```

Comment:
Only the hostname (and port) should be updated, not the other fields of the site object.